pillar.example

```diff
@@ -37,6 +37,10 @@ salt:
     salt_ssh: 'salt-ssh'
     pyinotify: 'python-pyinotify'  # the package to be installed for pyinotify
 
+  # Set which salt repository to use, default to https://repo.saltstack.com
+  # For older releases use http://archive.repo.saltproject.io
+  repo: 'https://archive.repo.saltproject.io'
```

Comment:
It should be http to not break tests

Reply:
I suppose this is why the previous code had mixed http and https url's. :disappointed: